src/plugins/github/declaration.js

```diff
@@ -85,7 +90,12 @@ const hasParentEdgeType = Object.freeze({
   defaultForwardWeight: 1,
   defaultBackwardWeight: 1 / 4,
   prefix: E.Prefix.hasParent,
-  description: "TODO: Add a description for this EdgeType",
+  description: dedent`\
+    Connects a GitHub entity to its child posts.
```

Comment:
Child posts only? Would a label not be a child of a repository?

Reply:
Updated for future proofing (although right now it's just posts).